src/_locales/nb/messages.json

```diff
@@ -82,11 +117,8 @@
 	"darkTheme": {
 		"message": "Mørkt"
 	},
-	"disableScrobblingDescription": {
-		"message": "Dersom dette valget er aktivert vil utvidelsen ikke automatisk scrobble det du ser på. Dette er nyttig om du bare ønsker å bruke sync av historikk."
```

Comment:
Do you remote the ability to disable scrobbling?

Reply:
Since you can now enable scrobbling separately for each streaming service, I don't see the point for a global option. You think there should be one?